blackboxopt/optimization_loops/dask_distributed.py

```diff
@@ -164,6 +174,11 @@ def run_optimization_loop(
                 break
 
         new_evaluations = dask_scheduler.check_for_results(timeout_s=20)
+
+        if post_evaluation_callback:
+            for e in new_evaluations:
+                post_evaluation_callback(e)
```

Comment:
What do you think about
```suggestion
        map(post_evaluation_callback, new_evaluations)
```
or
```suggestion
        _ = map(post_evaluation_callback, new_evaluations)
```
?

Reply:
I messed up this suggestion. I'll create a new comment.